src/utils.ts

```diff
@@ -319,7 +319,8 @@ export function computeQuantityRemaining(medicationRequestBundle): number {
         ...medicationRequestBundle.dispenses,
       ])
     ) {
-      throw new Error("Cannot calculate quantity remaining, units dont match");
+      console.error("Cannot calculate quantity remaining, units dont match");
```

Comment:
Don't think so, @mogoodrich.

Reply:
It seems this is the go-to and there a couple of uses like this in the code base. Will go ahead and get this in.

Reply:
If it's an issue a user should be able to fix, we should display a notification or highlight the field in question. If this is something the user can't influence, I suppose logging to the console is better than just swallowing an error. In this case, I would probably display an informative notification to the user.

Reply:
The user can't fix this, but an administrator good.  Ideally, I'd want to display a "contact your system administrator" error, but was just trying to get something in quickly, as hopefully this will be a edge case.